### ISSUE TITLE
missing cri-o-1.21 package for Ubuntu 20.04 from the repo?

### ISSUE THREAD

**Issue by kiwinesian** [cri-o/cri-o#5103]
**Description**

Looks like the cri-o package is missing from the `opensuse` repository?
was working fine until 2 hours ago.

http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable:/cri-o:/1.21/xUbuntu_20.04/

![image](https://user-images.githubusercontent.com/12476195/126218294-2c9a1f71-3f1a-4531-8a3c-4f5de6a14963.png)

**Steps to reproduce the issue:**
1. Go to http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable:/cri-o:/1.21/xUbuntu_20.04/
2. Can't find `cri-o`
3. Resulting `apt install cri-o` to fail

**Describe the results you received:**
```
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package cri-o
```

**Describe the results you expected:**
cri-o is install is successful

**Additional information you deem important (e.g. issue happens only occasionally):**

N/A

**Additional environment details (AWS, VirtualBox, physical, etc.):**
Cloud VM

**Comment by haircommander** [comment]
we don't package in the top-level anymore, because there are too many concurrently supported versions. I use this link to see the subprojects:
https://build.opensuse.org/project/subprojects/devel:kubic:libcontainers:stable

**Comment by haircommander** [comment]
possibly also related: i'm in the process of bumping the packages so there may be a delay between when they're bumped and when they're available for download

**Comment by kiwinesian** [comment]
ahh sounds good! thank youu sirr 🙏 
ill just wait until it is available again.

**Comment by kiwinesian** [comment]
do you mind to provide how to use https://build.opensuse.org/project/subprojects/devel:kubic:libcontainers:stable? 🙏 

been waiting for it to appear on the download file but hasn't appear yet.

**Comment by kiwinesian** [comment]
ended up using fix version 1.21.0
